insights/client/__init__.py

```diff
@@ -495,8 +492,7 @@ def get_registration_status(self):
             returns (json):
                 {'messages': [dotfile message, api message],
                  'status': (bool) registered = true; unregistered = false
-                 'unreg_date': Date the machine was unregistered | None,
-                 'unreachable': API could not be reached}
+                 'unreg_date': Date the machine was unregistered | None}
```

Comment:
This too is true only if _legacy_upload_ is True. [_insights.client.client.get_registration_status_](https://github.com/RedHatInsights/insights-core/blob/a1ffac37ba3e721dadaab6bb891c9800a8ef5a83/insights/client/client.py#L192) returns the result of [_insights.client.support.registration_check_](https://github.com/RedHatInsights/insights-core/blob/a1ffac37ba3e721dadaab6bb891c9800a8ef5a83/insights/client/support.py#L54), which returns
- either a result of [_insights.client.support.\_legacy\_registration\_check_](https://github.com/RedHatInsights/insights-core/blob/a1ffac37ba3e721dadaab6bb891c9800a8ef5a83/insights/client/support.py#L24), which is this dictionary (not JSON), if _legacy_upload_ is true,
- or result of [_insights.client.connection.InsightsConnection.api_registration_check_](https://github.com/RedHatInsights/insights-core/blob/a1ffac37ba3e721dadaab6bb891c9800a8ef5a83/insights/client/connection.py#L729) that is a boolean.